vnext/src/Libraries/Components/Button.windows.js

```diff
@@ -31,113 +31,113 @@ import type {PressEvent} from '../Types/CoreEventTypes';
 
 type ButtonProps = $ReadOnly<{|
   /**
-    Text to display inside the button. On Android the given title will be
+   Text to display inside the button. On Android the given title will be
     converted to the uppercased form.
-   */
+  */
   title: string,
```

Comment:
It would be ideal if we keep formatting consistent with the upstream file in RN core. We can otherwise get merge conflicts fir silly reasons if someone upstream were to update this documentation.